Comments/Services/ExportService.cs

```diff
@@ -63,7 +65,27 @@ public async Task<(IEnumerable<Models.Comment> comment, IEnumerable<Models.Answe
 			return (commentsInDB, answersInDB, questionsInDB, new Validate(true));
 	    }
 
-	    public (IEnumerable<Models.Comment> comment, IEnumerable<Models.Answer> answer, IEnumerable<Models.Question> question, Validate valid) GetAllDataForConsultationForCurrentUser(int consultationId)
+	    public (IEnumerable<Models.Comment> comment, IEnumerable<Models.Answer> answer, IEnumerable<Models.Question> question, Validate valid) GetDataSubmittedToLeadForConsultation(int consultationId)
+	    {
+		    var user = _userService.GetCurrentUser();
+		    var isLead = user.OrganisationsAssignedAsLead.Any();
+		    if (!isLead)
+		    {
+			    return (null, null, null, new Validate(valid: false, unauthenticated: true, message: $"User does not have access to download responses submitted to organisation leads."));
+			}
+
+		    var sourceURI = ConsultationsUri.CreateConsultationURI(consultationId);
+		    var commentsInDB = _context.GetCommentsSubmittedToALeadForURI(sourceURI);
+		    var answersInDB = _context.GetAnswersSubmittedToALeadForURI(sourceURI);
+		    var questionsInDB = new List<Question>();
```

Comment:
Consistency, this is the only place in the class that uses the declaration on line 11 using Question = Comments.Models.Question; all other references use Models.Question